cordova-plugin-mobilespec-tests/tests/datauri.tests.js

```diff
@@ -27,6 +27,7 @@ exports.defineAutoTests = function () {
 
         function onMessage(done, msg) {
             if (msg.data == 'foo') {
+                expect('Suppress SPEC HAS NO EXPECTATIONS').toBeDefined();
```

Comment:
Why don't the tests themselves have any `expect`s? Also, there are two identical `describe`s in this file. Is that expected?